helpers.json

```diff
@@ -1,502 +1,574 @@
+            {"ammo": {"type": "PowerBomb","count": 10}}
+          ],
+          "devNote": "1 Power bomb leniency per attempt, 10 leniency attempts."
+        }
+      ]
+    },
+    {
+      "name": "General",
+      "description": "Standard helpers used to simplify the game logic.",
+      "helpers": [
+        {
+          "name": "h_canNavigateHeatRooms",
+          "requires": [
+            {"or":[
+              "h_heatProof",
+              "canHeatRun"
+            ]}
+          ]
+        },
+        {
+          "name": "h_canNavigateUnderwater",
+          "requires": [
+            {"or":[
+              "Gravity",
+              "canSuitlessMaridia"
+            ]}
+          ]
+        },
+        {
+          "name": "h_canUseMorphBombs",
+          "requires": [
+            "Morph",
+            "Bombs"
+          ]
+        },
+        {
+          "name": "h_canUsePowerBombs",
+          "requires": [
+            "Morph",
+            {"ammo": { "type": "PowerBomb", "count": 1 }}
+          ]
+        },
+        {
+          "name": "h_canBombThings",
+          "requires": [
+            "Morph",
+            {"or": [
+              "Bombs",
+              {"ammo": { "type": "PowerBomb", "count": 1 }}
+            ]}
+          ]
+        },
+        {
+          "name": "h_canDestroyBombWalls",
+          "requires": [
+            {"or": [
+              "h_canBombThings",
+              "ScrewAttack"
+            ]}
+          ]
+        },
+        {
+          "name": "h_canIBJ",
+          "requires": [
+            "canIBJ",
+            "h_canUseMorphBombs"
+          ]
+        },
+        {
+          "name": "h_canFly",
+          "requires": [
+            {"or": [
+              "h_canIBJ",
+              "SpaceJump"
+            ]}
+          ]
+        },
+        {
+          "name": "h_canUseSpringBall",
+          "requires": [
+            "SpringBall",
+            "Morph"
+          ]
+        },
+        {
+          "name": "h_hasBeamUpgrade",
+          "requires": [
+            {"or": [
+              "Charge",
+              "Ice",
+              "Wave",
+              "Spazer",
+              "Plasma"
+            ]}
+          ]
+        },
+        {
+          "name": "h_canBlueGateGlitch",
+          "requires": [
+            "canGateGlitch",
+            {"or": [
+              {"ammo": { "type": "Missile", "count": 1 }},
+              {"ammo": { "type": "Super", "count": 1 }}
+            ]},
+            "h_BlueGateGlitchLeniency"
+          ]
+        },
+        {
+          "name": "h_canGreenGateGlitch",
+          "requires": [
+            "canGateGlitch",
+            {"ammo": { "type": "Super", "count": 1 }},
+            "h_GreenGateGlitchLeniency"
+          ]
+        },
+        {
+          "name": "h_canHeatedBlueGateGlitch",
+          "requires": [ 
+            "h_canNavigateHeatRooms",
+            {"heatFrames": 60},
+            "canGateGlitch",
```

Comment:
Either h_canBlueGateGlitch
or h_BlueGateGlitchLeniency

Reply:
I think this should be correct as is. It has its own leniency a few lines down. If I call `h_canBlueGateGlitch`, it will use the `h_BlueGateGlitchLeniency` as well - giving 3x ammo and 2x heat frames

Reply:
I must have been thinking that `h_heated...` was only giving heat frames